modules/scripting/mod-script-pipe/PipeServer.cpp

```diff
@@ -161,6 +161,7 @@ void PipeServer()
    if (fromFifo == NULL)
```

Comment:
Just a quick suggestion: I noticed you're using NULL in the code. Since we're working with C++11 and beyond, it might be a good idea to switch to nullptr. It's more type-safe and helps avoid some subtle bugs that can come from using NULL, especially in template or overloaded function contexts. Plus, nullptr has its own type (std::nullptr_t), which makes it more explicit and clear. Just a small modern C++ improvement!
 There is allot of technical literature on this topic.
 
 1. **Effective Modern C++** by Scott Meyers  
2. **The C++ Programming Language (4th Edition)** by Bjarne Stroustrup  
3. **C++ Primer (5th Edition)** by Stanley B. Lippman, Josée Lajoie, and Barbara E. Moo  
4. **More Effective C++** by Scott Meyers

Reply:
I'm at work right now but I can change it to nullptr when I get home.